src/hello.js

```diff
@@ -0,0 +1,2 @@
+const hello = () => "i don't trust you because you aren't javaScript";
+export default hello;
```

Comment:
- [ ] Kindly remove this file because it is not serving any necessary role in your database.